samples/java_layout/src/main/java/io/customer/android/sample/java_layout/ui/settings/SettingsActivity.java

```diff
@@ -75,6 +75,10 @@ private void parseLinkParams() {
         // https://www.java-sample.com/settings&site_id=xxx&api_key=yyy
```

Comment:
if you are updating the deep link to include `cdp_api_key`, maybe we need to add it here as well for reference?

Reply:
Good catch, updated